### PR TITLE
feat: add env var to skip local import

### LINKED ISSUE #2543: importLocal check may invoke wrong version of WDS in a Yarn workspace
- Operating System: Ubuntu 16
- Node Version: 12.16.1
- NPM Version: 6.13.4
- webpack Version: 4.43.0
- webpack-dev-server Version: 3.11.0
- Browser: N/A

- [x] This is a **bug**
- [x] This is a **modification** request

### Code

Folder structure (including installed packages):

```
- yarn.lock
- node_modules // workspace root
  - import-local
  - webpack-dev-server@3.7.2
- my-project
  - webpack.config.js
  - package.json // includes an npm script to invoke WDS
  - node_modules
    - webpack-dev-server@3.11.0
```

### Expected Behavior

When I invoke WDS from the nested `my-project` directory it should use v3.11.0

### Actual Behavior

v3.11.0 is invoked correctly at first, but the `importLocal` check in the `/bin/webpack-dev-server.js` script causes v3.7.2 to be used. This is because of a bug in the `import-local` package (which I have reported [here](https://github.com/sindresorhus/import-local/issues/7)) where it doesn't take into account that there might be multiple `node_modules` folders, like in a Yarn workspace.

The bug is intermittent because it depends on Yarn's hoisting behaviour; if `import-local` had been placed in the nested `node_modules` directory then v3.11.0 would be used as intended.

The bug can occur for any versions of WDS; 3.11.0 and 3.7.2 are just the versions in my project.

### For Bugs; How can we reproduce the behavior?

It's tricky to reproduce this reliably as it depends on Yarn's hoisting behaviour, but it should be possible to manually recreate a folder structure like the above and observe the behaviour.

### For Features; What is the motivation and/or use-case for the feature?

I would like to propose a flag to skip the `importLocal` check. In my scenario I can be sure there won't be a global installation of WDS, so there's no need to run the check at all, which would avoid any issues like this.

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**
feat

**Did you add tests for your changes?**
Yes

**Summary**

Add env var to skip import-local

**Does this PR introduce a breaking change?**
No

**Other information**
Fix #2543

### CODE REVIEW

File: test/build/import-local/import-local.test.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+describe('import local', () => {
+    beforeEach(() => {
+        importLocalMock.mockClear();
+    });
+    it('should skip import local when supplied', () => {
+        const { exitCode, stderr, stdout } = run(__dirname, [], {
+            env: { WEBPACK_CLI_SKIP_IMPORT_LOCAL: true },
+        });
+        console.log({ exitCode, stderr, stdout });
```
Comment (alexander-akait) [inline test/build/import-local/import-local.test.js:16]: Let's remove it

Comment (anshumanv) [reply test/build/import-local/import-local.test.js]: hm done

**Comment by alexander-akait** [comment]
After merge we need improve docs

**Comment by anshumanv** [comment]
will improve

**Review by alexander-akait — APPROVED** [review]
/cc @webpack/cli-team

**Review by snitin315 — APPROVED** [review]

**Pull request merged**